src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -316,4 +305,147 @@ private Human humanOf(String forename, String forename2, String surname) {
     }
 
 
+    public List<uk.gov.justice.digital.delius.data.api.ResponsibleOfficer> responsibleOfficersOf(Offender offender, boolean current) {
+
+        ArrayList<ResponsibleOfficer> responsibleOfficers = new ArrayList<>();
+
+        if (offender.getOffenderManagers() != null && !offender.getOffenderManagers().isEmpty()) {
+
+            for (uk.gov.justice.digital.delius.jpa.standard.entity.OffenderManager om : offender.getOffenderManagers()) {
+
+                boolean isCurrentRo   = (om.getResponsibleOfficer() != null && om.getResponsibleOfficer().getEndDate() == null);
+                boolean isCurrentOm   = (om.getActiveFlag().intValue() == 1 && om.getEndDate() == null);
```

Comment:
Consider pulling these out into functions and testing independently?

Reply:
Done.